src/tests/components/NewArticleForm.test.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import NewArticleForm from '../../components/Articles/NewArticleForm';
+
+describe('NewArticleForm component', () => {
+  const mockStore = configureMockStore();
+  let wrapper;
+  const handleSubmit = jest.fn();
+  const resetForm = jest.fn();
+  const handleChange = jest.fn();
+  const handleEditorChange = jest.fn();
+  const title = '';
+  const description = '';
+  const body = '';
+  const loading = false;
+
+  beforeEach(() => {
+    mockStore({});
+    wrapper = shallow(<NewArticleForm
+      handleSubmit={handleSubmit}
+      resetForm={resetForm}
+      handleChange={handleChange}
+      handleEditorChange={handleEditorChange}
+      body={body}
+      title={title}
+      description={description}
+      loading={loading}
+    />);
+  });
+
+  it('should render correctly', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should call resetForm when the clear button is clicked', () => {
+    wrapper.find('#clear-button').simulate('click', { preventDefault() {} });
+    expect(resetForm).toBeCalled();
```

Comment:
Add a test showing what describe `resetForm ` does.

Reply:
This was handled in the file `ViewArticle.test.js`